src/main/scala/info/bethard/timenorm/formal/Readers.scala

```diff
@@ -136,7 +136,10 @@ class AnaforaReader(val DCT: Interval)(implicit data: Data) {
       case ("Two-Digit-Year", Some(value), N, N, N, None) => value.partition(_ != '?') match {
         case (year, questionMarks) => YearSuffix(interval(properties), year.toInt, questionMarks.length)
       }
-      case ("Between", None, N, N, N, None) => Between(interval(properties, "Start-"), interval(properties, "End-"))
+      case ("Between", None, N, N, N, None) => interval(properties, "Start-").end == interval(properties, "End-").start match {
```

Comment:
You should check the `Start-Included` and `End-Included` properties, not test whether the end and start are equal.